.github/workflows/integration_test.yaml

```diff
@@ -1,13 +1,12 @@
 name: Integration tests
 
-on: [pull_request, workflow_call]
+on:
+  pull_request:
 
 jobs:
-  test-with-secrets:
```

Comment:
The last discussion with @yanksyoon was that we would leave these in. What is the reason for removing them please?

Reply:
Those will be added on a follow-up PR so that PRs depending on this one can start being merged